daemon/mgr/cri_test.go

```diff
@@ -99,6 +172,37 @@ func TestToCriSandboxState(t *testing.T) {
 
 // Container related unit tests.
 
+func TestParseContainerName(t *testing.T) {
+	format := fmt.Sprintf("%s_${container name}_${sandbox name}_${sandbox namespace}_${sandbox uid}_${attempt times}", kubePrefix)
```

Comment:
We've tested `parseContainerName` in `TestContainerNameRoundTrip`, so this test function may not be needed.